README.md

```diff
@@ -66,61 +65,51 @@ When using Object options you can add to each option a className string to furth
 The `className` prop is passed down to the wrapper `div`, which also has the `Dropdown-root` class.
 
 ```JavaScript
-<Dropdown className='myClassName' />;
+<DropdownNOW className='myClassName' />;
 ```
 
 **controlClassName**
 
 The `controlClassName` prop is passed down to the control `div`, which also has the `Dropdown-control` class.
 
 ```JavaScript
-<Dropdown controlClassName='myControlClassName' />;
+<DropdownNOW controlClassName='myControlClassName' />;
 ```
 
 **placeholderClassName**
 
 The `placeholderClassName` prop is passed down to the placeholder `div`, which also has the `Dropdown-placeholder` class.
 
 ```JavaScript
-<Dropdown placeholderClassName='myPlaceholderClassName' />;
+<DropdownNOW placeholderClassName='myPlaceholderClassName' />;
 ```
 
 **menuClassName**
 
 The `menuClassName` prop is passed down to the menu `div` (the one that opens and closes and holds the options), which also has the `Dropdown-menu` class.
 
 ```JavaScript
-<Dropdown menuClassName='myMenuClassName' />;
+<DropdownNOW menuClassName='myMenuClassName' />;
 ```
 
 **arrowClassName**
 
 The `arrowClassName` prop is passed down to the arrow `span` , which also has the `Dropdown-arrow` class.
 
 ```JavaScript
-<Dropdown arrowClassName='myArrowClassName' />;
+<DropdownNOW arrowClassName='myArrowClassName' />;
```

Comment:
can keep the component var name as Dropdown

Reply:
ok :) I like the yelling NOW but if you don't agree I'll remove it

Reply:
Dropdown is restored and DropdownNOW is removed